packages/frontend-defaults/report.api.md

```diff
@@ -43,4 +44,17 @@ export interface CreateAppOptions {
 export function createPublicSignInApp(options?: CreateAppOptions): {
   createRoot(): React_2.JSX.Element;
 };
+
+// @public (undocumented)
+export function discoverAvailableFeatures(config: Config): {
+  features: FrontendFeature[];
```

Comment:
Oooh, I realized this one should probably support feature loaders too right?

Reply:
Should it?  I don't think it explicitly works right now, since `discoveredFeatures` doesn't get passed through feature loader resolution logic.  I guess we could?  ...But is it event possible for `window['__@backstage/discovered__']` to be populated with async feature loaders?

Reply:
Yep right now we don't support it, but I'm thinking we'd want to prepare consumers of this function to support it. I'm a little bit torn on whether it's a good idea to discover feature loaders, but it is how it works in the backend system currently so thinking we should stick to that.

Reply:
I'll do that when rebasing my PR on top of this one.

Reply:
Alright let's add this after

Reply:
I'm doing it right now while rebasing my PR on top of this one.